internal/lexer/token.go

```diff
@@ -0,0 +1,51 @@
+package lexer
+
+type TokenType int
+
+const (
+	EOF TokenType = iota
+	IDENTITY
+	NUMBER
+	STRING
+
+	PATH     // .
+	PIPE     // |
+	LBRACKET // [
+	RBRACKET // ]
+
+	beginOperatorSection
+	ASSIGN   // =
+	EQUALITY // ==
+	MERGE    // +
+	endOperatorSection
+)
+
+var tokens = []string{
+	EOF:      "EOF",
+	IDENTITY: "IDENTITY",
+	NUMBER:   "NUMBER",
+	STRING:   "STRING",
+
+	PATH:     "PATH",
+	PIPE:     "PIPE",
+	LBRACKET: "LBRACKET",
+	RBRACKET: "RBRACKET",
+
+	ASSIGN:   "ASSIGN",
```

Comment:
Don't we call it replace? I believe this should aligned with the design doc.

Reply:
Done